### ISSUE TITLE
 Create Pipenv for scripters to install dependencies easily

### ISSUE THREAD

**Issue by makuto** [makuto/Liked-Saved-Image-Downloader#52]
See [here](https://packaging.python.org/tutorials/managing-dependencies/).

**Comment by Amar1729** [comment]
Saw this issue, but I think a more viable alternative could be [poetry](https://python-poetry.org/) - in general, I've found it to be far faster and generally nicer than `pipenv`. I put together a `pyproject.toml` (below) that works for me with this project (and could open a PR if you like).

After cloning, installation of dependencies is done via `poetry install`, and the project can be used either with `poetry run <command>` or executing `poetry shell` and then running a command (e.g. `python LikedSavedDownloaderServer.py`).

```
[tool.poetry]
name = "liked-saved-image-downloader"
version = "0.9.0"
description = "Save content you enjoy!"
authors = ["makuto <macoymadson@gmail.com>"]
license = "MIT"

[tool.poetry.dependencies]
python = "^3.8"
praw = "^7.1.0"
pytumblr = "^0.1.0"
jsonpickle = "^1.4.1"
tornado = "^6.0.4"
youtube-dl = "^2020.7.28"
gfycat = { git = "https://github.com/ankeshanand/py-gfycat", branch = "master" }
pixivpy = { git = "https://github.com/upbit/pixivpy", branch = "master" }
passlib = {version = "^1.7.2", optional = true}
bcrypt = {version = "^3.2.0", optional = true}
argon2_cffi = {version = "^20.1.0", optional = true}
imgurpython = "^1.1.7"

[tool.poetry.dev-dependencies]

[build-system]
requires = ["poetry>=0.12"]
build-backend = "poetry.masonry.api"
```

**Comment by makuto** [comment]
This is now merged thanks to your PR #82 . Thanks!
